apps/client/src/app/pages/portfolio/fire/fire-page.html

```diff
@@ -106,14 +106,17 @@ <h4 class="align-items-center d-flex">
         <ng-container>&nbsp;</ng-container>
         <ng-container i18n>and a safe withdrawal rate (SWR) of</ng-container>
         <ng-container>&nbsp;</ng-container>
-        <span class="font-weight-bold"
-          ><gf-value
-            class="d-inline-block"
-            [isPercent]="true"
-            [locale]="user?.settings?.locale"
-            [precision]="1"
-            [value]="user?.settings?.safeWithdrawalRate" /></span
-        >.
+        <select
+          class="d-inline-block font-weight-bold w-auto swr-select"
+          [disabled]="
```

Comment:
Could you disable the form control in the component (see `this.userService.stateChanged`)